ui-library-filesharing-chat-composite/README.md

```diff
@@ -0,0 +1,68 @@
+# Azure Communication Services application with file sharing
+
+This is a complete sample showcasing integration of file sharing feature
+into an [Azure Communication Services][docs-root] powered [React] application.
+
+For a step-by-step guide to building this sample yourself, see the associated tutorial at MSDN.
+
+    TODO: Link to tutorial once available.
```

Comment:
Are we planning to write a tutorial that goes with this sample?

Reply:
It's already written here
https://github.com/MicrosoftDocs/azure-docs-pr/pull/194148

This sample will be linked to that tutorial

Reply:
@prabhjot-msft This is a chicken and egg situation.
The tutorial is not published yet because it doesn't have the sample linked in it.
And we are leaving a placeholder for the tutorial here.
What comes first?

Reply:
<3. Great to see a tutorial for this.

The placeholder is fine. I just wanted to make sure this isn't a copy-pasta leftover.

Reply:
Yes, we have documentation ready. Once merged will add the link